games-util/pyfa/pyfa-1.35.2.ebuild

```diff
@@ -0,0 +1,101 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+PYTHON_COMPAT=( python2_7 )
+PYTHON_REQ_USE="sqlite,threads"
+
+inherit eutils gnome2-utils python-r1
+
+DESCRIPTION="Python Fitting Assistant - a ship fitting application for EVE Online"
+HOMEPAGE="https://github.com/pyfa-org/Pyfa"
+
+LICENSE="GPL-3+ LGPL-2.1+ CC-BY-2.5 free-noncomm"
+SLOT="0"
+if [[ ${PV} = 9999 ]]; then
+	EGIT_REPO_URI="https://github.com/pyfa-org/Pyfa.git"
+	inherit git-r3
+	KEYWORDS="x86"
+else
+	SRC_URI="https://github.com/pyfa-org/Pyfa/archive/v${PV}.tar.gz -> pyfa-${PV}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+IUSE="+graph"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+RDEPEND="dev-python/python-dateutil[${PYTHON_USEDEP}]
+	>=dev-python/sqlalchemy-1.0.5[${PYTHON_USEDEP}]
+	dev-python/wxpython:3.0[${PYTHON_USEDEP}]
+	>=dev-python/logbook-1.0.0[${PYTHON_USEDEP}]
+	dev-python/requests[${PYTHON_USEDEP}]
+	graph? (
+		dev-python/matplotlib[wxwidgets,${PYTHON_USEDEP}]
+		dev-python/numpy[${PYTHON_USEDEP}] )
+	${PYTHON_DEPS}"
+DEPEND="app-arch/zip"
+
+[[ ${PV} = 9999 ]] || S=${WORKDIR}/Pyfa-${PV}
+
+src_prepare() {
+	# get rid of CRLF line endings introduced in 1.1.10 so patches work
+	edos2unix config.py pyfa.py gui/bitmapLoader.py service/settings.py
+
+	# load gameDB and images from separate staticdata directory
+	eapply "${FILESDIR}/${PN}-1.33.1-staticdata.patch"
+
+	# fix import path in the main script for systemwide installation
+	eapply "${FILESDIR}/${PN}-1.33.1-import-pyfa.patch"
+
+	eapply_user
+
+	touch __init__.py
```

Comment:
??

Reply:
The intention of the application developers is that pyfa is installed somewhere in the users home directory. The ebuild however installs it in the proper locations for python packages and since there is no __init__.py coming with the sources one is created to make python recognize it as a proper package.

The ebuild and program works without this, but I think it is the right thing to do here: https://docs.python.org/3/tutorial/modules.html#packages

Reply:
Just `||die`, please.

Reply:
I'd also suggest adding a comment so that future maintainers aren't confused with this line.